classes/game.rb

```diff
@@ -6,6 +6,7 @@ def initialize(title, multiplayer, last_played_at, publish_date, archived)
     @title = title
     @multiplayer = multiplayer
     @last_played_at = last_played_at
+    @author = nil
```

Comment:
There is no need to set the **author** to **nil** here because it is already set to null by the initialize method of Item, and you are calling that on line 5

Reply:
Actually, it's this that sorted the errors from my end